build.gradle

```diff
@@ -71,7 +77,7 @@ dependencies {
   testImplementation "org.apache.avro:avro-compiler:1.12.0"
 
   // Forcing logback to be used for test logging
-  testImplementation "ch.qos.logback:logback-classic:1.3.14"
+  testImplementation "ch.qos.logback:logback-classic:1.5.18"
```

Comment:
This requires Java 17 - which is fine if you're going in that direction anyway - i.e. forcing marklogic-junit5 to use Spring 6.x.

Reply:
I think that forcing Java 17 for testing is fine. So, yes, that's the plan.